blocks/header/header.js

```diff
@@ -1,21 +1,87 @@
 import { readBlockConfig, decorateIcons } from '../../scripts/lib-franklin.js';
 
+const mobileBreakpoint = 992;
+let globalWindowWidth = window.innerWidth;
```

Comment:
we have a different approach in https://github.com/hlxsites/netcentric/blob/main/blocks/leader-profiles/leader-profiles.js#L4 should we make this a util config? Same for the breakpoint widths.

Reply:
That's a good idea, I can create a PR to extract a util function into `scripts.js`